apps/web/templates/about.html

```diff
@@ -0,0 +1,213 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block menu %}
+<!--==========================
+    Menu
+============================-->
+<nav id="nav-menu-container">
+  <ul class="nav-menu">
+    <li><a href="#body">Home</a></li>
+    <li class="menu-has-children menu-active"><a href="">SBG-RJ</a>
+      <ul>
+        <li><a href="#">Sobre</a></li>
+        <li><a href="#">Eventos</a></li>
+        <li><a href="#">Notícias</a></li>
+        <li><a href="#">Produtos</a></li>
+      </ul>
+    </li>
+    <li><a href="#">Publicações</a></li>
+    <li><a href="#">Galeria</a></li>
+    <li><a href="#">Regionais</a></li>
+    <li><a href="#">Contato</a></li>
```

Comment:
Lembra de adicionar as urls nos links todos @pedrocecchetti .

```suggestion
    <li><a href="{% url 'web:contact' %}">Contato</a></li>
```